ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -403,6 +404,24 @@ void HydroMechanicsProcess<GlobalDim>::initializeConcreteProcess(
             MeshLib::MeshItemType::Node, 1);
         mesh_prop_nodal_p->resize(mesh.getNumberOfNodes());
         _process_data.mesh_prop_nodal_p = mesh_prop_nodal_p;
+
+        _process_data.nodal_forces = MeshLib::getOrCreateMeshProperty<double>(
```

Comment:
Minor naming inconsistency: cf. ` _process_data.mesh_prop_nodal_p` and `_process_data.nodal_forces`.

Reply:
Btw: In HM you do this initialization directly in the ctor.

Reply:
Made consistent.
In HM it is better, but for historical reasons it is initialized later in LIE/{SD,HM} (and requires a `const_cast` on the mesh too).